### ISSUE TITLE
Theme breaks grav

### ISSUE THREAD

**Issue by rustbuckett** [paulmassen/grav-theme-chalk#18]
I installed a fresh grav, went to the basedir and ran `bin/gpm install chalk`. It successfully installed all of the dependencies and the sample content. I activated the theme in the Admin tool and the html rendering breaks. This is what I see:

```
<link href="/gravbase/user/themes/chalk/css-compiled/light.css" type="text/css" rel="stylesheet"> <link href="https://cdnjs.cloudflare.com/ajax/libs/fluidbox/2.0.5/css/fluidbox.min.css" type="text/css" rel="stylesheet"> <link href="/gravbase/user/plugins/markdown-notices/assets/notices.css" type="text/css" rel="stylesheet"> <link href="/gravbase/user/plugins/form/assets/form-styles.css" type="text/css" rel="stylesheet"> <link href="/gravbase/user/plugins/highlight/css/dracula.css" type="text/css" rel="stylesheet"> <link href="/gravbase/user/plugins/login/css/login.css" type="text/css" rel="stylesheet">
Grav

    Home
    Blog
    Tags

<h1>Say Hello to Grav!</h1> <h2>installation successful...</h2> <p>Congratulations! You have installed the <strong>Base Grav Package</strong> that provides a <strong>simple page</strong> and the default <strong>Quark</strong> theme to get you started.</p> <div class="notices red"> <p>If you see a <strong>404 Error</strong> when you click <code>Typography</code> in the menu, please refer to the <a href="http://learn.getgrav.org/troubleshooting/page-not-found">troubleshooting guide</a>.</p> </div> <h3>Find out all about Grav</h3> <ul> <li>Learn about <strong>Grav</strong> by checking out our dedicated <a href="http://learn.getgrav.org">Learn Grav</a> site.</li> <li>Download <strong>plugins</strong>, <strong>themes</strong>, as well as other Grav <strong>skeleton</strong> packages from the <a href="http://getgrav.org/downloads">Grav Downloads</a> page.</li> <li>Check out our <a href="http://getgrav.org/blog">Grav Development Blog</a> to find out the latest goings on in the Grav-verse.</li> </ul> <div class="notices blue"> <p>If you want a more <strong>full-featured</strong> base install, you should check out <a href="http://getgrav.org/downloads"><strong>Skeleton</strong> packages available in the downloads</a>.</p> </div> <h3>Edit this Page</h3> <pre><code>ck out our [Grav Development Blog](http://getgrav.org/blog) to find out the latest goings on in the Grav-verse. !!! If you wan</code></pre> <p>To edit this page, simply navigate to the folder you installed <strong>Grav</strong> into, and then browse to the <code>user/pages/01.home</code> folder and open the <code>default.md</code> file in your <a href="http://learn.getgrav.org/basics/requirements">editor of choice</a>. You will see the content of this page in <a href="http://learn.getgrav.org/content/markdown">Markdown format</a>.</p> <h3>Create a New Page</h3> <p>Creating a new page is a simple affair in <strong>Grav</strong>. Simply follow these simple steps:</p> <ol> <li> <p>Navigate to your pages folder: <code>user/pages/</code> and create a new folder. In this example, we will use <a href="http://learn.getgrav.org/content/content-pages">explicit default ordering</a> and call the folder <code>03.mypage</code>.</p> </li> <li> <p>Launch your text editor and paste in the following sample code:</p> <pre><code>--- title: My New Page --- # My New Page! This is the body of **my new page** and I can easily use _Markdown_ syntax here.</code></pre> </li> <li> <p>Save this file in the <code>user/pages/03.mypage/</code> folder as <code>default.md</code>. This will tell <strong>Grav</strong> to render the page using the <strong>default</strong> template.</p> </li> <li> <p>That is it! Reload your browser to see your new page in the menu.</p> </li> </ol> <div class="notices yellow"> <p>NOTE: The page will automatically show up in the Menu after the "Home" menu item. If you wish to change the name that shows up in the Menu, simple add: <code>menu: My Page</code> between the dashes in the page content. This is called the YAML front matter, and it is where you configure page-specific options.</p> </div>

<script src="/gravbase/system/assets/jquery/jquery-2.x.min.js"></script> <script src="//cdnjs.cloudflare.com/ajax/libs/jquery-throttle-debounce/1.1/jquery.ba-throttle-debounce.min.js"></script> <script src="/gravbase/user/themes/chalk/js/vendor.js"></script> <script src="/gravbase/user/themes/chalk/js/scrollappear.js"></script> <script src="https://cdnjs.cloudflare.com/ajax/libs/fluidbox/2.0.5/js/jquery.fluidbox.min.js"></script> <script src="/gravbase/user/themes/chalk/js/application.js"></script> <script src="/gravbase/user/plugins/highlight/js/highlight.pack.js"></script> <script> hljs.initHighlightingOnLoad(); </script> 
```

This is **not** the page source even though that's kinda what it looks like. If I load the page source, is shows the normal tags and head information, but then it shows all of the `&gt;` and `&lt;` characters in the source.

I love this theme and really want to use it, but I don't understand what's happening when I activate it.

**Comment by xer0-1ne** [comment]
This theme is broken for me too.

It doesn't great Grav, but it doesn't have any style, it appears to be basic and unthemed.

**Comment by icyavocado** [comment]
I created a pull request to fix this issue #19 . Hopefully this is the right fix.

**Comment by paulmassen** [comment]
Fixed, thanks @icyavocado
